packages/core/src/Asset/NftMetadata/fromMetadatum.ts

```diff
@@ -101,11 +95,10 @@ const getAssetMetadata = (policy: Cardano.MetadatumMap, assetName: Cardano.Asset
       })()
   );
 
-// TODO: consider hoisting this function together with cip25 types to core or a new cip25 package
 /**
  * @returns {NftMetadata | null} CIP-0025 NFT metadata
```

Comment:
Is this docblock stale?

Reply:
This is part of LW-6909, no need to have this inline comment

Reply:
I'm referring to the docblock that's reading "CIP-0025 NFT metadata" but the commit indicates it's being generalised, so should it be updated?

Reply:
Updated doc comments in d87885f66f26df44543c8059cd9c6fe936c9cf1d and cb282ec5acb64f37f87ebab961b25e0d519edf08